elephant/spike_train_generation.py

```diff
@@ -1060,5 +1064,86 @@ def compound_poisson_process(rate, A, t_stop, shift=None, t_start=0 * ms):
                 for cp in cpp]
             return cpp
 
+
 # Alias for the compound poisson process
 cpp = compound_poisson_process
+
+
+def homogeneous_poisson_process_with_refr_period(rate, refr_period=3. * ms,
+                                                 t_start=0.0 * ms,
+                                                 t_stop=1000.0 * ms,
+                                                 as_array=False):
+    """
+    Returns a spike train whose spikes are a realization of a Poisson process
+    with the given rate and refractory period starting at time `t_start` and
+    stopping time `t_stop`.
+
+    All numerical values should be given as Quantities, e.g. 100*Hz.
+
+    Parameters
+    ----------
+
+    rate : Quantity scalar with dimension 1/time
+           The rate of the discharge.
+    refr_period : Quantity scalar with dimension time
+                  The time period the after one spike no other spike is
+                  emitted.
```

Comment:
I think you should specify everywhere the default parameters. Btw: is 3ms appropriate?